app/build.gradle

```diff
@@ -116,12 +116,14 @@ dependencies {
     implementation "com.google.dagger:hilt-android:$hilt_version"
     kapt "com.google.dagger:hilt-compiler:$hilt_version"
 
-    implementation 'org.briarproject:dont-kill-me-lib:0.2.5'
+    implementation 'org.briarproject:dont-kill-me-lib:0.2.6'
```

Comment:
I think this should get pulled in by onionwrapper-android. Maybe we can use this as a test to investigate the issue of onionwrapper-core not being pulled in automatically?

Reply:
If I remove this line, I get a compile error: `Unresolved reference: AndroidWakeLockManagerImpl`

Reply:
Looks like we should be using `compile` rather than `runtime` for the dependencies in the onionwrapper POM:

> scope:
> This element refers to the classpath of the task at hand (compiling and runtime, testing, etc.) as well as how to limit the transitivity of a dependency. There are five scopes available:
> 
> * compile - this is the default scope, used if none is specified. Compile dependencies are available in all classpaths. Furthermore, those dependencies are propagated to dependent projects.
> * provided - this is much like compile, but indicates you expect the JDK or a container to provide it at runtime. It is only available on the compilation and test classpath, and is not transitive.
> * runtime - this scope indicates that the dependency is not required for compilation, but is for execution. It is in the runtime and test classpaths, but not the compile classpath.
> * test - this scope indicates that the dependency is not required for normal use of the application, and is only available for the test compilation and execution phases. It is not transitive.
> * system - this scope is similar to provided except that you have to provide the JAR which contains it explicitly. The artifact is always available and is not looked up in a repository.

https://maven.apache.org/pom.html#Dependencies

Maybe your changes in onionwrapper!4 (changing `implementation` to `api`) already cause the POM to list the dependencies as `compile` rather than `runtime`?

Reply:
Looks like onionwrapper!4 does change the dependencies to `compile`. :thumbsup: